### DISCUSSION TITLE
Cannot update child in the Question set

### DISCUSSION THREAD

**Discussion by siddhi0719** [Sunbird-coKreat/Community#95] [General Discussions]
@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

I've created a Question set and which is currently live.

Following are the steps that I've followed to update the Live Question set:

Made changes in the Child section and was able to successfully sent it for review.
Tried to review the Question set from the Reviewer's workspace, not able to see the updated changes made in the child section.
I tried to check the network panel also, there is no API getting failed.

The above issue is been solved for 5.0 version we need to add the patch for 4.6 version.
Please let me know who can help me to add this patch.

**Comment by Aparna-karri591** [comment]
@siddhi0719 Please Refer this https://github.com/orgs/Sunbird-coKreat/discussions/75, It may help you.

**Reply by siddhi0719** [reply]
@Aparna-karri591 Please tell me where I can find the patch because I was unable to locate it.

**Reply by Aparna-karri591** [reply]
@siddhi0719 Please post it in Inquiry BB https://github.com/orgs/Sunbird-inQuiry/discussions
